src/pages/builders/challenges.tsx

```diff
@@ -1,17 +1,81 @@
-import React from 'react'
+import React, { useMemo } from 'react'
 import type { GetServerSideProps, NextPage } from 'next'
+import moment from 'moment'
+import Phases from '@/data/phase'
+import { Accordion } from '@/components/index'
+import { useAccordion } from '@/hook/useAccordion'
+import { ContentBlock, taskContentIcon } from '../validators/tasks'
 import { config } from '@/lib/config'
 import type { Config } from '@/types/config.type'
+import type { TaskContent } from '@/data/phase/dto.type'
 
 export type ChallengesProps = Pick<Config, 'urls' | 'phases'>
 
-const Challenges: NextPage<ChallengesProps> = () => {
+const Challenges: NextPage<ChallengesProps> = props => {
+  const { phases } = props
+  const challenges = useMemo(
+    () => Phases(phases)[Phases(phases).length - 1].challenges ?? [],
+    [phases]
+  )
+  const [activeIndex, setActiveIndex] = useAccordion()
+
+  const handleClick = (index: number) => () => {
+    activeIndex === index ? setActiveIndex(null) : setActiveIndex(index)
+  }
+
   return (
     <div className="okp4-nemeton-web-page-main">
       <div className="okp4-nemeton-web-page-content-container challenges" id="challenges">
         <div className="okp4-nemeton-web-page-challenges-container">
           <div className="okp4-nemeton-web-page-challenges-content">
             <h1>Challenges</h1>
+            <div className="okp4-nemeton-web-page-divider" />
+            {challenges.map(({ group: challengePart, challengeDescription, tasks }) => (
+              <div key={challengePart}>
+                <h2>{challengePart}</h2>
+                {challengeDescription}
+                {tasks.map(({ taskName, taskContent, taskDuration }, index) => {
```

Comment:
Please, do not use `tasks` or anything related into `challenges` page.

Keep in mind that they could be completely different in terms of types and structures (dto)